src/measures.py

```diff
@@ -43,22 +43,20 @@ def __init__(self, client, addresses):
         self.socket.setblocking(0)
         self.time = functools.partial(_TimeContext, self.socket, client, addresses)
 
-    def count(self, metric, counter=1, dimensions=None):
+    def count(self, metric, counter=1, dimensions={}):
```

Comment:
@EwertonBello it is dangerous to set argument as a empty dict, we have a full explanation here: https://stackoverflow.com/questions/26320899/why-is-the-empty-dictionary-a-dangerous-default-value-in-python

Reply:
@wpjunior Neste caso específico, utiliza-se update `dimensions.update(message)`, não é acumulativo.

```
def f(value, key, hash={}):
    hash.update({'k':key})
    return hash
```
output
```
print(f('a', 1))
{'k': 1}
print(f('b', 2))
{'k': 2}
```

No exemplo problemático exposto no artigo:

```
def f(value, key, hash={}):
    hash[value] = key
    return hash
```
output
```
print(f('a', 1))
{'a': 1}
print(f('b', 2))
{'a': 1, 'b': 2}
```